### ISSUE TITLE
question about loading 

### ISSUE THREAD

**Issue by WallcroftUK** [ApourtArtt/NostaleWidget#34]
hello first i want to ask if you have discord or anything for better chatting 

second is that if its possible to chain load like:
First.EWS that load Second.dll that load EWSF.dll

thats all thank you for reply

**Comment by ApourtArtt** [comment]
Hello

> first i want to ask if you have discord or anything for better chatting

Sorry, I am not helping through Discord

> second is that if its possible to chain load like:
First.EWS that load Second.dll that load EWSF.dll

Yes, you can definitely do it, but in order to first inject First.EWS, you willneed to modify NostaleClientX.exe as, by default, EWSF.EWS is loaded.

**Comment by WallcroftUK** [comment]
so first i need load First.EWS that will load others right

**Comment by ApourtArtt** [comment]
Yes. You can look at this example: https://github.com/xAfterLife/EWSF
Which shows how to load/use EWSF.dll from EWSF.EWS
